docs/tutorials/transforms.ipynb

```diff
@@ -429,13 +436,39 @@
     "print(x.dtype, x.min(), x.max())"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Appending Indices with `torchgeo.transforms`\n",
+    "\n",
+    "`torchgeo.transforms` support appending indices to a specified channel dimension. The following transforms are available:\n",
+    "\n",
+    "- **AppendBNDVI**\n",
+    "- **AppendGBNDVI**\n",
+    "- **AppendGNDVI**\n",
+    "- **AppendGRNDVI**\n",
+    "- **AppendNBR**\n",
+    "- **AppendNDBI**\n",
+    "- **AppendNDRE**\n",
+    "- **AppendNDSI**\n",
+    "- **AppendNDVI**\n",
+    "- **AppendNDWI**\n",
+    "- **AppendNormalizedDifferenceIndex**\n",
+    "- **AppendRBNDVI**\n",
+    "- **AppendSWI**\n",
+    "- **AppendTriBandNormalizedDifferenceIndex**\n",
```

Comment:
Not sure if it's worth it to list all of these out. I would instead list a couple indices that TorchGeo knows how to compute and then link to the full API docs.

Reply:
done!